src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -257,48 +292,72 @@ private void attachCaseByCcdId(
         Long exceptionRecordId,
         String idamToken,
         String userId,
-        CaseDetails exceptionRecord
+        CaseDetails exceptionRecordData,
+        boolean ignoreWarnings
     ) {
         log.info("Attaching exception record '{}' to a case by CCD ID '{}'", exceptionRecordId, targetCaseCcdRef);
 
-        CaseDetails theCase = ccdApi.getCase(targetCaseCcdRef, exceptionRecordJurisdiction);
-        List<Map<String, Object>> targetCaseDocuments = getScannedDocuments(theCase);
-
-        verifyExceptionRecordIsNotAttachedToCase(exceptionRecordJurisdiction, exceptionRecordId);
-
-        //This is done so exception record does not change state if there is a document error
-        checkForDuplicatesOrElse(
-            exceptionRecordDocuments,
-            targetCaseDocuments,
-            ids -> throwDuplicateError(targetCaseCcdRef, ids)
-        );
-
-        List<Map<String, Object>> newCaseDocuments = attachExceptionRecordReference(
-            exceptionRecordDocuments,
-            exceptionRecordId
-        );
+        Validation<String, Classification> classificationValidation =
+            hasJourneyClassificationForAttachToCase(exceptionRecordData);
+        if (classificationValidation.isValid()) {
+            if (SUPPLEMENTARY_EVIDENCE_WITH_OCR == classificationValidation.get()) {
+                ServiceConfigItem serviceConfigItem = getServiceConfig(exceptionRecordData);
+                Validation<Seq<String>, ExceptionRecord> exceptionRecordValidation =
+                    exceptionRecordValidator.getValidation(exceptionRecordData);
+                if (exceptionRecordValidation.isValid()) {
+                    ccdCaseUpdater.updateCase(
+                        exceptionRecordValidation.get(),
+                        serviceConfigItem,
+                        ignoreWarnings,
+                        idamToken,
+                        userId,
+                        targetCaseCcdRef
+                    );
+                } else {
+                    throw new CallbackException(exceptionRecordValidation.getError().mkString());
```

Comment:
If exception record is invalid, the caseworker should see the related errors. This exception will result in a 500 response returned to ccd an error being logged in Bulk Scanning and the caseworker seeing a generic error message.